src/Action/AddAction.php

```diff
@@ -50,10 +51,8 @@ class AddAction extends BaseAction
         'viewVar' => null,
         'relatedModels' => true,
         'entityKey' => 'entity',
-        'saveOptions' => [
-            'validate' => true,
-            'atomic' => true
-        ],
+        'entityOptions' => [],
+        'saveOptions' => [],
```

Comment:
It is actually a good practice to pass the same options to patchEntity and save. So having a single array should be fine